plugins/plugin-docker/che-plugin-docker-client/src/main/java/org/eclipse/che/plugin/docker/client/InitialAuthConfig.java

```diff
@@ -43,14 +43,17 @@
 
     AuthConfig predefinedConfig;
 
+    private String serverAddress = "https://index.docker.io/v1/";
```

Comment:
Why you decided that server address is docker hub? Some vendors would like to use private registry instead